docs/hsa/hsa.rst

```diff
@@ -0,0 +1,178 @@
+
+This will show the available tables in HSA TAP service in the Herschel Science Archive.
+
+-------------------------------------
+6. Getting columns details of HSA TAP
+-------------------------------------
+
+.. code-block:: python
+
+  >>> from astroquery.hsa import HSA
+  >>>
+  >>> HSA.get_columns('hsa.v_active_observation')
+  INFO: Retrieving tables... [astroquery.utils.tap.core]
+  INFO: Parsing tables... [astroquery.utils.tap.core]
+  INFO: Done. [astroquery.utils.tap.core]
+  ['aor', 'bii', 'dec', 'duration', 'end_time', 'fov', 'global_science_area', 'icon_image', 'icon_location', 'image_2_5_location', 'image_location', 'ingest_queue_oid', 'instrument_oid', 'is_active_version', 'is_public', 'lii', 'naif_id', 'num_publications', 'observation_id', 'observation_oid', 'observer', 'observing_mode_oid', 'obsstate', 'od_number', 'pa', 'polygon_fov', 'position', 'prop_end', 'proposal_id', 'quality_report_location', 'ra', 'science_area', 'science_category', 'spg_id', 'start_time', 'status', 'target_name', 'urn_version']
+
+This will show the column details of the table 'hsa.v_active_observation' in HSA TAP service in the Herschel Science Archive.
+
+-------------------------------------
+7. Procedure example
+-------------------------------------
+
+First retrieve the observation IDs based on a position on the sky. To achive this, query the TAP service.
+
+.. code-block:: python
+
+  >>> from astroquery.hsa import HSA
+  >>>
+  >>> HSA.query_hsa_tap("select top 10 observation_id from hsa.v_active_observation where contains(point('ICRS', hsa.v_active_observation.ra, hsa.v_active_observation.dec),circle('ICRS', 100.2417,9.895, 1.1))=1", output_format='csv', output_file='results.cvs')
```

Comment:
We should have a wrapper function implemented to do this, e.g.:

```python
def query_observations(coordinate, radius, n_obs=10):
     ...
```

to match the general astroquery user API.